ui/app/src/modules/Content/Delete/DeleteDialog.tsx

```diff
@@ -273,14 +314,20 @@ function DeleteDialogWrapper(props: DeleteDialogProps) {
     );
   }, [selectedItems, setApiState, siteId]);
 
+  useEffect(() => {
+    setSubmissionCommentCount({
+      currentLength: submissionComment.length
```

Comment:
is there a reason to not use directly the comment to get it's length in place? that way we save some mem by dropping an additional state and effect.